install_config/configuring_sdn.adoc

```diff
@@ -0,0 +1,111 @@
+= Configuring the SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The link:../architecture/additional_concepts/sdn.html[OpenShift SDN] enables
+communication between pods across the OpenShift cluster, establishing a _pod
+network_. Two link:../architecture/additional_concepts/sdn.html[SDN plug-ins]
+are currently available (*ovs-subnet* and *ovs-multitenant*), which provide
+different methods for configuring the pod network.
+
+For initial link:../install_config/install/advanced_install.html[advanced
+installations], the *ovs-subnet* plug-in is installed and configured by default,
+though it can be
+link:../install_config/install/advanced_install.html#configuring-ansible[overridden
+during installation] using the `*os_sdn_network_plugin_name*` parameter.
+
+ifdef::openshift-enterprise[]
+For initial link:../install_config/install/quick_install.html[quick
+installations], the *ovs-subnet* plug-in is installed and configured by default
+as well, and can be reconfigured post-installation.
+endif::[]
+
+[[configuring-the-pod-network-on-masters]]
+== Configuring the Pod Network on Masters
+
+Cluster administrators can control pod network settings on masters by modifying
+parameters in the `*networkConfig*` section of the
+link:../install_config/master_node_configuration.html[master configuration file]
+(located at *_/etc/origin/master/master-config.yaml_* by default):
+
+====
+[source,yaml]
+----
+networkConfig:
+  clusterNetworkCIDR: 10.1.0.0/16 <1>
+  hostSubnetLength: 8 <2>
+  networkPluginName: "redhat/openshift-ovs-subnet" <3>
+  serviceNetworkCIDR: 172.30.0.0/16 <4>
+----
+<1> Cluster network for node IP allocation
+<2> Number of bits for pod IP allocation within a node
+<3> Set to *redhat/openshift-ovs-subnet* for the flat SDN plug-in or
+*redhat/openshift-ovs-multitenant* for the multi-tenant SDN plug-in
```

Comment:
Maybe "<3> Set to *redhat/openshift-ovs-subnet* for the *ovs-subnet* plug-in or *redhat/openshift-ovs-multitenant* for the *ovs-multitenant* plug-in to be consistent with the terms used everywhere else? (Likewise below.)